Form endpoint

SajaLahaleeh opened the pull request.

SajaLahaleeh committed:
solve conflict between two branch

SajaLahaleeh committed:
solve conflect

jema28 commented on config.env:
```diff
@@ -0,0 +1 @@
+DATABASE_URL = postgres://yizthhwjzzmgrm:8d830f4e30e56aedc270b92fe7fdf98d083113b2169c99394e52d4a17ba4a808@ec2-50-19-222-129.compute-1.amazonaws.com:5432/d9bltlsuv2jqs
```
config.env is up again!

jema28 commented on src/controllers/POSTpainting-form.js:
```diff
@@ -0,0 +1,24 @@
+(req, res) => {
+  let data = "";
+  const paint = req.body;
+  // console.log(req.body, "01010101010");
```
Please delete these console logs.

jema28 commented on src/controllers/POSTpainting-form.js:
```diff
@@ -0,0 +1,24 @@
+    paint.price,
+    paint.description,
+    paint.img,
+    // paint.artistName,
```
And this comment.

jema28 commented on src/queries/addPainting.js:
```diff
@@ -0,0 +1,9 @@
+const dbConnection = require("../database/db_connection.js");
+const text = `INSERT INTO paintings (paintingname, price, description, img, id_artist) VALUES($1,$2,$3,$4, $5);`;
+// const text = `INSERT INTO paintings (name, price, description, img, id_artist) VALUES($1,$2,$3,$4, (select id from artists where name ilike $5) );`;
```
Delete this!

jema28 commented on src/views/painting-form.hbs:
```diff
@@ -0,0 +1,28 @@
+    <label for="paintName">Paint Name</label>
+    <input type="text" name="paintName" id="paintName">
+    <br>
+    {{!-- <label for="artistName">Artisi Name</label> --}}
```
Delete these! all comments.

jema28 commented on src/views/partials/header.hbs:
```diff
@@ -1,3 +1,3 @@
 <header>
-    <img src="/public/assest/back1.jpg" alt="">
+    <img src="/assest/back1.jpg" class="back" alt="background">
```
Asset is spelt wrong: 'Asset'.

SajaLahaleeh committed:
Delete config.env

jema28 commented on src/queries/getData.js:
```diff
@@ -1,7 +1,9 @@
+  )
   .then(res => res.rows)
-  .catch(e => e)
+  .catch(e => e);
```
You can write console log in the catch `.catch(console.log)

jema28 commented on src/queries/getAllArtists.js:
```diff
@@ -0,0 +1,7 @@
+const getAllArtists = dbConnection
+  .query(`select * from artists ;`)
+  .then(res => res.rows)
+  .catch(e => e);
```
Same here, look at comment on catch.

jema28 reviewed (changes requested):
Please fix your eslint.

SajaLahaleeh committed:
remove config.env

SajaLahaleeh committed:
Merge branch 'formEndpoint' of github.com:fack2/Artists8 into formEndpoint

SajaLahaleeh committed:
eslintify files and delete comments

SajaLahaleeh committed:
eslint all files / add eslint script files

SajaLahaleeh committed:
delete hbs comment in form

SajaLahaleeh committed:
fix assets spelling

SajaLahaleeh committed:
fix spelling in painting form

SajaLahaleeh committed:
add dev script

SajaLahaleeh committed:
refactor router and controller files
- reaarange router index.
- merge get and post painting form
- delete endpoints and eslint
- add spaces in files

jema28 reviewed (approved):
Beautiful PR!
